modules/sugar/src/test/ts/browser/LocationTest.ts

```diff
@@ -136,7 +136,8 @@ UnitTest.asynctest('LocationTest', (success, failure) => {
     pos = SugarLocation.viewport(body);
     Assert.eq('', 0, pos.top);
     Assert.eq('', 0, pos.left);
-    Assert.eq('scroll bar width, got=' + scrollBarWidth, true, scrollBarWidth > 5 && scrollBarWidth < 50 || (platform.os.isMacOS() && scrollBarWidth === 0));
+    const noVisibleScrollbarBrowser = platform.os.isMacOS() || (platform.browser.isFirefox() && platform.os.isLinux());
```

Comment:
This doesn't fix it for Firefox on Windows 11 as it also has no scrollbar. Should that be added now as well?

Reply:
I don't have any way to test that do we have Windows 11 VMs somewhere?

Reply:
There is a build nodes for windows 11 yes... it needs you to either SSH though or use the `bedrock-windows-11` label instead from what I know.

Reply:
I think adding this should solve it though:
```ts
(platform.os.isWindows() && platform.os.version.major >= 11)
```

Reply:
I guess I could just add it now and then if it breaks things on Win 11 we figure it out later.

Reply:
Oops sorry Spocke, I stuffed up my previous example as that would apply to all browsers. We just want firefox. I'll leave some suggestions.

Reply:
Ooops I should have read that code better as well.

Reply:
Doing code changes while on a meeting and listening one one ear is probably not a good this. Will fix this now.

Reply:
Oh bummer it seems that UA in FF on Win 11 is still 10. So we can't sniff out Windows 11 it seems.
I created a follow up ticket here: https://ephocks.atlassian.net/browse/TINY-9203 so make some form of feature detection. I added it as a training ticket since it seems small in scope and only affects tests.